inspirehep/modules/records/serializers/json.py

```diff
@@ -64,6 +64,31 @@ def get_display_fields(record):
 class JSONBriefSerializer(JSONSerializer):
     """JSON brief format serializer."""
 
+    def serialize_search(self, pid_fetcher, search_result, links=None,
+                         item_links_factory=None):
+        """Serialize a search result.
+
+        :param pid_fetcher: Persistent identifier fetcher.
+        :param search_result: Elasticsearch search result.
+        :param links: Dictionary of links to add to response.
+        """
+        import json
+        from flask import get_flashed_messages
```

Comment:
Do these imports have to be local? If yes, please add a comment about it, otherwise move them to the outermost scope.